exampleCourse/questions/element/matching/question.html

```diff
@@ -150,3 +150,45 @@
         </pl-matching>
     </div>
 </div>
+
+<div class="card my-2">
+    <div class="card-body">
+        <pl-question-panel>
+            <p>
+                Example 8: This example uses the default "right" placement for the options.
+            </p>
+        </pl-question-panel>
+            
+        <pl-matching answers-name="ex8" options-placement="right">
+            <pl-statement match="Ottawa">Canada</pl-statement>
+            <pl-statement match="Washington, D.C.">United States</pl-statement>
+            <pl-statement match="Mexico City">Mexico</pl-statement>
+            <pl-statement match="Paris">France</pl-statement>
+            
+            <pl-option>Tijuana</pl-option>
+            <pl-option>Toronto</pl-option>
+            <pl-option>New York City</pl-option>
+        </pl-matching>
+    </div>
+</div>
+
+<div class="card my-2">
+    <div class="card-body">
+        <pl-question-panel>
+            <p>
+                Example 9: This example uses the "bottom" placement for the options.
+            </p>
+        </pl-question-panel>
+            
+        <pl-matching answers-name="ex9" options-placement="bottom">
+            <pl-statement match="Ottawa">Canada</pl-statement>
+            <pl-statement match="Washington, D.C.">United States</pl-statement>
+            <pl-statement match="Mexico City">Mexico</pl-statement>
+            <pl-statement match="Paris">France</pl-statement>
+            
+            <pl-option>Tijuana</pl-option>
+            <pl-option>Toronto</pl-option>
+            <pl-option>New York City</pl-option>
+        </pl-matching>
```

Comment:
A description of when this placement is useful (e.g., long statements or options) might be useful here. Even better, if the example itself has a use-case that looks better in bottom placement would be ideal. Here is a suggestion:

```suggestion
        <pl-question-panel>
            <p>
                Example 8: This example uses the "bottom" placement for the options. This attribute is useful if the statements or options are longer, for better visualization.
            </p>
        </pl-question-panel>
            
        <pl-matching answers-name="ex8" options-placement="bottom">
            <pl-statement match="Hamlet">To be, or not to be, that is the question.</pl-statement>
            <pl-statement match="Macbeth">Tomorrow, and tomorrow, and tomorrow, creeps in this petty pace from day to day, to the last syllable of recorded time.</pl-statement>
            <pl-statement match="Mark Antony">Friends, Romans, countrymen, lend me your ears; I come to bury Caesar, not to praise him.</pl-statement>
            <pl-statement match="Jaques">All the world’s a stage, and all the men and women merely players: they have their exits and their entrances; and one man in his time plays many parts, his acts being seven ages.</pl-statement>
            
            <pl-option>Henry V</pl-option>
            <pl-option>Romeo</pl-option>
            <pl-option>Richard II</pl-option>
        </pl-matching>
```